pkg/operator/starter.go

```diff
@@ -58,6 +58,7 @@ func RunOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 		operatorclient.OperatorNamespace,
 		operatorclient.TargetNamespace,
 		"kube-system",
+		"openshift-infra",
```

Comment:
Which resources are getting watched in this namespace? Anything new popped up in this namespace? Or, has it been forgotten?